templates/all-services/all-services-en.html

```diff
@@ -49,755 +49,101 @@ <h2 class="h3 mb-4">Most requested services</h2>
+			<h3><a href="https://www.canada.ca/en/services/business.html">Business and Industry</a></h3>
+			<p>Starting a business, permits, copyright, business support, selling to government</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/benefits.html">Benefits</a></h3>
+			<p>"(<abbr title="Employment Insurance">EI<abbr/>)", family and sickness leave, child benefit, pensions, housing, student aid, disabilities, after a death</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/health.html">Health</a></h3>
+			<p>Food, nutrition, diseases, vaccines, drugs, product safety and recalls</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/taxes.html">Taxes</a></h3>
+			<p>Income tax, payroll, GST/HST, contribution limits, tax credits, charities</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/environment.html">Environment and natural resources</a></h3>
+			<p>Weather, climate, agriculture, wildlife, pollution, conservation, fisheries</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/defence.html">National security and defence</a></h3>
+			<p>Military, transportation and cyber security, securing the border, counter-terrorism</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/culture.html">Culture. history and sport</a></h3>
+			<p>Arts, media, heritage, official languages, national identity and funding</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/policing.html">Policing, justice and emergencies</a></h3>
+			<p>Safety, justice system, prepare for emergencies, services for victims of crime</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/transport.html">Transport and infrastructure</a></h3>
+			<p>Aviation, marine, road and rail, car seat and vehicle recalls</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="http://international.gc.ca/world-monde/index.aspx?lang=eng">Canada and the world</a></h3>
+			<p>Foreign policy, trade agreements, development work, global issues</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/finance.html">Money and finances</a></h3>
+			<p>Personal finance, credit reports, fraud protection, paying for education, housing</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/science.html">Science and innovation</a></h3>
+			<p>Scientific research on health, environment and space, grants and funding</p>
+		</div>
+		<div class="col-md-6 col-lg-4">
+			<h3><a href="https://www.canada.ca/en/services/life-events.html">Manage life events</a></h3>
+			<p>Benefits and resources to help navigate life's biggest transitions</p>
+		</div>
```

Comment:
Please move the bottom section inside the above container. Please make sure every div is closed.